client/my-sites/domains/domain-management/add-google-apps/add-email-addresses-card.jsx

```diff
@@ -128,20 +133,56 @@ const AddEmailAddressesCard = createReactClass( {
 		);
 	},
 
-	allEmailAddressFieldsets() {
+	renderFieldsets() {
+		return this.state.fieldsets.map( ( _, index ) => {
```

Comment:
Oh, what is this `_` symbol? 😮

I suspect that's just a way to denote the fact that we don't use the current item being iterated on, right?

Reply:
It was already in the code.  https://github.com/Automattic/wp-calypso/pull/29155/files/6453262add9bfe9bb941d81e17407bff974be99c#diff-4755603c8aafd8730708bb8f5ef5d6e7L134

I think it is just a placeholder to get the second prop, index.